src/main/java/tasks/Deadline.java

```diff
@@ -0,0 +1,66 @@
+package tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import exceptions.DukeException;
+
+
+/** A class that functions as an abstraction of a deadline task. */
+public class Deadline extends Task {
+
+    public static final String WRONG_FORMAT_ERROR_STRING =
+            "Format for deadlines: 'deadline [some task] /by [dd/mm/yyyy-hh:mm]'";
+    private static final DateTimeFormatter INPUT_FORMAT = DateTimeFormatter.ofPattern("dd/MM/yyyy-HH:mm");
+    private static final DateTimeFormatter PRETTY_FORMAT = DateTimeFormatter.ofPattern("MMM dd yyyy HH:mm");
+
+    private final Task.TaskType taskType = Task.TaskType.DEADLINE;
+    private final String taskName;
+    private final LocalDateTime deadline;
```

Comment:
As they are constants, shouldn't they be in upper case with underscores separating different words?